test/test_target.py

```diff
@@ -465,6 +465,23 @@ def test_opencl_emits_ternary_operators_correctly(ctx_factory, target):
     assert out_dict["y3"] == 128
 
 
+def test_scalar_array_take_offset(ctx_factory):
+    ctx = ctx_factory()
+    cq = cl.CommandQueue(ctx)
+
+    knl = lp.make_kernel(
+        "{:}",
+        """
+        y = 2*x
+        """,
+        [lp.GlobalArg("x", shape=(), dtype=np.float64, offset=lp.auto),
+         ...])
+
+    knl = lp.set_options(knl, "write_cl")
+    evt, (out,) = knl(cq, x=np.asarray(21, dtype=np.float64))
```

Comment:
Could you make this an actual CL array with an offset? (I.e. maybe an `arange`, and pass the 21st entry?)

Reply:
Done!  b72b88c